rdbms/src/main/java/io/spine/server/storage/jdbc/query/ColumnAwareWriteQuery.java

```diff
@@ -41,6 +48,27 @@
         return columnTypeRegistry;
     }
 
+    /**
+     * Obtains the name-to-index transformer for the specified record.
+     *
+     * @param record           the record to extract column names
+     * @param firstColumnIndex the index of the first entity column in a query
+     * @return a {@code Function} transforming a column name to the its index
+     * @see io.spine.server.entity.storage.ColumnRecords#feedColumnsTo
+     */
+    Function<String, Integer> getTransformer(EntityRecordWithColumns record,
+                                             int firstColumnIndex) {
+        final Map<String, Integer> result = new HashMap<>();
```

Comment:
Please use Guava for consistency with the rest of code.

Also this has to be an immutable map at the end. So I would use a builder for the `ImmutableMap`.

Reply:
Done.